src/connection/adapter/base_http_adapter.ts

```diff
@@ -294,17 +294,30 @@ export abstract class BaseHttpAdapter implements Connection {
   }
 
   private logResponse(
+    request: Http.ClientRequest,
     params: RequestParams,
     response: Http.IncomingMessage,
     startTimestamp: number
   ) {
+    const getRequestHeaders = () => {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const { authorization, ...headers } = request.getHeaders()
+      return headers
+    }
+    const getRequestBody = () => {
+      return typeof params.body === 'string' ? params.body : 'Stream'
```

Comment:
I'm not sure it's safe to log the body as it might contain sensitive data to insert into ClickHouse. Let's remove it.

Reply:
I'd suggest maybe adding a possibility to enable or disable it in the logger, along with the fields whitelist, cause it might be useful for debugging purposes.